### PR TITLE
settle event now emits actual debt settled rather than t0 amount

### PR DESCRIPTION
**Pull request by EdNoepel**
## Description

Emit settled debt in actual amount rather than t0 amount.
Also added some `tearDown` modifiers where missing from affected unit tests.

## Purpose

All other auction events emit amounts in actual terms.  This was brought to our attention by BlockAnalytica.

## Impact

No change in pool contract sizes.  Tiny gas cost for the multiplication.

## Tasks

- [x] Changes to protocol contracts are covered by unit tests executed by CI.
- [x] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [ ] Scope labels have been assigned as appropriate.
- [ ] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

File: tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol [hunk 1]
```diff
@@ -970,14 +970,14 @@ contract ERC721PoolCollateralTest is ERC721HelperContract {
             from:        _lender,
             borrower:    _borrower,
             maxDepth:    11,
-            settledDebt: 102.277919151483596286 * 1e18
+            settledDebt: 402.720322053719854868 * 1e18
```
Comment (EdNoepel) [inline tests/forge/unit/ERC721Pool/ERC721PoolCollateral.t.sol:973]: This test skips 27 years after debt is drawn, explaining the difference here and line 980.

File: tests/forge/unit/ERC721Pool/ERC721PoolLiquidationsSettleAuction.t.sol [hunk 2]
```diff
@@ -160,7 +160,7 @@ contract ERC721PoolLiquidationsSettleAuctionTest is ERC721HelperContract {
             from:        _lender,
             borrower:    _borrower,
             maxDepth:    1,
-            settledDebt: 2485.445445980762364312 * 1e18
+            settledDebt: 5_000.732078876477988054 * 1e18
```
Comment (EdNoepel) [inline tests/forge/unit/ERC721Pool/ERC721PoolLiquidationsSettleAuction.t.sol:163]: `setUp()` skips 14 years, explaining the large changes in these tests.

**Review by MikeHathaway — APPROVED** [review]
LGTM - still seems to be a test failure in `test_regression_settle_with_reserves`

**Review by ith-harvey — APPROVED** [review]
LGTM

**Comment by EdNoepel** [comment]
> LGTM - still seems to be a test failure in `test_regression_settle_with_reserves`

Indeed; fixed in 0c42ada89c0c85a62c5dc6cc402000f46aaf0e0a.

**Comment by grandizzy** [comment]
proposed slightly improvement https://github.com/ajna-finance/contracts/pull/1005/commits/b554d431f8ba36c8d8747ef753a90ebe31244ddb

**Pull request merged**